.github/workflows/build.yml

```diff
@@ -124,11 +127,21 @@ jobs:
           PACKAGE_URL: honnef.co/go/tools/cmd/staticcheck
           PACKAGE_VERSION: ${{ steps.setup-env.outputs.staticcheck-version }}
         run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
-      - name: Install Terraform-docs
-        env:
-          PACKAGE_URL: github.com/terraform-docs/terraform-docs
-          PACKAGE_VERSION: ${{ steps.setup-env.outputs.terraform-docs-version }}
-        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+      # We are temporarily using @mcdonnnj's forked branch of terraform-docs
+      # until his PR: https://github.com/terraform-docs/terraform-docs/pull/745
+      # is approved. This temporary fix will allow for ATX header support when
+      # terraform-docs is run during linting.
```

Comment:
Please create an issue in this repo about changing from @mcdonnnj's fork back to the main `terraform-docs` repo after https://github.com/terraform-docs/terraform-docs/pull/745 is (hopefully) merged.  Once the issue has been created, please update the code comment here to be a `TODO` and include a link to that issue.  See [here](https://github.com/cisagov/cool-assessment-provisioner-iam/blob/3235e1f2db56423c52bb13f8460436ffb9d74de4/locals.tf#L46-L50) for an example.

Reply:
The comment and issue both look great, thanks!  FYI, I made a couple of small formatting edits to #165.